docs/src/pages/components/data-grid/rendering/VisibleToolbarGrid.tsx

```diff
@@ -11,7 +11,13 @@ export default function VisibleToolbarGrid() {
 
   return (
     <div style={{ height: 300, width: '100%' }}>
-      <DataGrid {...data} showToolbar />
+      <DataGrid
+        {...data}
+        components={{
+          Toolbar: GridToolbar,
+        }}
+        showToolbar
```

Comment:
I assume it's no longer needed, proving a component to the slot take this purpose. 

```suggestion
```

Reply:
We still do, the change where this is no longer needed is in the other PR https://github.com/mui-org/material-ui-x/pull/948. We decided to do it incrementally as far as I remember.

Reply:
OK, my bad